README.md

```diff
@@ -84,6 +84,12 @@ To take advantage of distributed computation of explanations, install `alibi` wi
 pip install alibi[ray]
 ```
 
+For SHAP support, install `alibi` as follows:
+```bash
+pip install alibi
+pip install alibi[shap]
```

Comment:
May be wrong here but, isn't `pip install alibi[shap]` installing alibi + alibi with shap dependencies already?

Reply:
Yes it would, but the one-command results in the broken installation because `shap` is built against the newest `numpy` if no `numpy` is present in the environment...